packages/venia-concept/src/components/Navigation/__tests__/categoryTree.spec.js

```diff
@@ -177,7 +177,7 @@ test('child node correctly sets new root and parent ids', async () => {
         </MockedProvider>
     );
 
-    await wait();
+    await wait(200);
```

Comment:
this change was added to fix flakiness where it wouldn't finish resolving in time for the assertion